src/stapi_fastapi/routers/root_router.py

```diff
@@ -84,6 +90,22 @@ def __init__(
             tags=["Orders"],
         )
 
+        self.add_api_route(
+            "/orders/{order_id}/statuses",
+            self.get_order_statuses,
+            methods=["GET"],
+            name=f"{self.name}:list-order-statuses",
+            tags=["Orders"],
+        )
+
+        self.add_api_route(
```

Comment:
eventually, I think it would be good to have a more modular structure whereby users could choose to implement this endpoint or not.